### PR TITLE
Add heading 1 component

### PR DESCRIPTION
**Pull request by jreis13**
Why:
* Need to create all the typography components.

How:
* Add and style the heading 1 component, using variables from constant file.

### CODE REVIEW

File: frontend/src/components/Typography/Head1/index.module.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.head1 {
+  font-family: var(--font-family-body);
+  font-size: var(--font-size-lg);
+  font-weight: 500;
+  line-height: 40px;
+}
```
Comment (joaojusto) [inline frontend/src/components/Typography/Head1/index.module.css:6]: Aren't we missing `color` here? 🤔

File: frontend/src/components/Typography/Head1/index.module.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.head1 {
```
Comment (joaojusto) [inline frontend/src/components/Typography/Head1/index.module.css:1]: Would you mind keeping `heading1` to be consitent with design naming?

**Review by joaojusto — APPROVED** [review]
@jreis13 This is good, just take a look at my comments but feel free to merge as soon as you tackle them

File: frontend/src/components/Typography/Head1/index.module.css (hunk 1 above)
Comment (jreis13) [reply frontend/src/components/Typography/Head1/index.module.css]: Added

File: frontend/src/components/Typography/Head1/index.module.css (hunk 2 above)
Comment (jreis13) [reply frontend/src/components/Typography/Head1/index.module.css]: Makes sense, changed in this as well as in the other heading components.

**Pull request merged**